src/AxisAlignedBB.php

```diff
@@ -229,43 +183,20 @@ public function trimmedCopy(Facing $face, float $distance) : AxisAlignedBB{
 	 *
 	 * @return $this
 	 */
-	public function stretch(Axis $axis, float $distance) : AxisAlignedBB{
-		if($axis === Axis::Y){
-			$this->minY -= $distance;
-			$this->maxY += $distance;
-		}elseif($axis === Axis::Z){
-			$this->minZ -= $distance;
-			$this->maxZ += $distance;
-		}elseif($axis === Axis::X){
-			$this->minX -= $distance;
-			$this->maxX += $distance;
-		}
-
-		return $this;
-	}
-
-	/**
-	 * Returns a stretched copy of this bounding box.
-	 * @see AxisAlignedBB::stretch()
-	 */
 	public function stretchedCopy(Axis $axis, float $distance) : AxisAlignedBB{
-		return (clone $this)->stretch($axis, $distance);
+		return match($axis){
+			Axis::Y => new AxisAlignedBB($this->minX, $this->minY - $distance, $this->minZ, $this->maxX, $this->maxY + $distance, $this->maxZ),
+			Axis::Z => new AxisAlignedBB($this->minX, $this->minY, $this->minZ - $distance, $this->maxX, $this->maxY, $this->maxZ + $distance),
+			Axis::X => new AxisAlignedBB($this->minX - $distance, $this->minY, $this->minZ, $this->maxX + $distance, $this->maxY, $this->maxZ)
+		};
```

Comment:
The original code was more readable. Better to convert it to use local vars and then `return new AxisAlignedBB(...)` at the end.